src/components/Slideover/slideover.scss

```diff
@@ -0,0 +1,53 @@
+.sb-slideover {
+  width: 100%;
+  max-width: 62.4rem;
+  height: 100vh;
+  position: fixed;
+  background-color: $white;
+  box-sizing: border-box;
+  padding: 2.5rem;
+  transform: translateX(100%);
+  overflow: auto;
+  
+  &-open {
+    animation-timing-function: linear;
+    animation-direction: normal;
+    animation-name: slidein;
+    animation-duration: .2s;
+    transform: translateX(0%);
+    opacity: 1;
+    right: 0;
+    top: 0;
+  }
+
+  &__close-button {
+    position: fixed;
+    right: 2.5rem;
+    float: right;
+    border: none;
+    background-color: #FFF;
+    cursor: pointer;
+    padding: 0;
+  }
+
+  .sb-modal-footer {
+    padding: 0;
+    position: absolute;
+    box-sizing: border-box;
+    margin-left: -25px;
+    padding-right: 2.5rem;
+    border-radius: 0px;
+  }
+
+  .sb-button:not(:first-child) {
+    margin-left: 2rem;
+  }
+}
+
+@keyframes slidein {
```

Comment:
how about create a file with just animations and transitions?

Reply:
Good idea man